woocommerce/woocommerce
# A11y improvement to the variable product add to cart table on product singles

### All Submissions:


### Changes proposed in this Pull Request:

The current variable product add to cart table on product singles does not use table headers or define a role for the table. This PR changes the attribute label column from a `td` to a `th`. 

To preserve the appearance of the `td` default browser styling, a `text-align: left;` property is added to the `label` class.

Additionally, there is a minor code quality change to alphabetize the order of the properties within the selector.

### Reason for change:

Tables can be used in two methods; data or layout. Data tables allow screen reader users to understand column and row relationships while layout tables read cells as a series of unrelated paragraphs with no tabular structure. Without a `th` or `role`, screen readers apply heuristics to decide whether a table is a layout table or data table. These heuristics vary greatly between screen readers and are affected by the browser being used, window size, and font size so the outcome is very unpredictable without a `th` or `role` applied. 

Because this table's intent is to relate the `label` column to the `value` column, the `td` should be replaced with a `th` element.

The style change is applied because the default `th` text alignment for browsers like Chrome is `center`. By setting this to `text-align: left;`, the label should remain left aligned as it was when the containing element was a `td`.

### How to test the changes in this Pull Request:

1. View a variable product on the front-end.
2. Inspect the attribute label to confirm it is contained within a `th` element.
3. Confirm that the attribute label remains left aligned.

### Other information:


### Changelog entry

Improve accessibility of the variable product add to cart table on product singles.

### FOR PR REVIEWER ONLY:
